addons/project/views/project_task_views.xml

```diff
@@ -801,7 +803,6 @@
             </field>
         </record>
 
-
```

Comment:
remove

Reply:
Hello
i have done the changes. I hope it's fine now